### PR TITLE
fix doxygen errors

### PR DESCRIPTION
**Pull request by cbielow**
this fixes quite a lot of broken links in our documenation, especially with TOPP/UTILS

I also tried to switch to MARKDOWN support, but accidental use of indentation will break a lot of our pages since the markdown interpreter runs first and converts these parts to verbatim blocks...
If at some point this is fixed, we could use

| potential predecessor tools   |    MapAlignerTreeGuided    | potential successor tools |
| ----------------------------- | -------------------------- | --------------------------  |
| TOPP_IDMapper                 | ^                          | TOPP_FeatureLinkerUnlabeledKD |

as table format

### CODE REVIEW

**Comment by cbielow** [comment]
rebuild jenkins

**Comment by jpfeuffer** [comment]
Seems unrelated.

**Comment by jpfeuffer** [comment]
feel free to merge when travis passes.

File: src/openms/include/OpenMS/ANALYSIS/QUANTITATION/ProteinResolver.h [hunk 1]
```diff
@@ -164,50 +164,50 @@ namespace OpenMS
...
-      @brief NOT IMPLEMENTED YET
-
-      @param protein_nodes
-      @param peptide_nodes
-      @param reindexed_proteins
-      @param reindexed_peptides
-      @param peptide_identifications
-      @param output
-    */
+    // /**
```
Comment (timosachsenberg) [inline src/openms/include/OpenMS/ANALYSIS/QUANTITATION/ProteinResolver.h]: is this correct?

File: src/openms/source/APPLICATIONS/ToolHandler.cpp [hunk 2]
```diff
@@ -107,8 +107,6 @@ namespace OpenMS
     tools_map["NoiseFilterGaussian"] = Internal::ToolDescription("NoiseFilterGaussian", "Signal processing and preprocessing");
     tools_map["NoiseFilterSGolay"] = Internal::ToolDescription("NoiseFilterSGolay", "Signal processing and preprocessing");
     tools_map["OMSSAAdapter"] = Internal::ToolDescription("OMSSAAdapter", "Identification");
-    tools_map["OpenPepXL"] = Internal::ToolDescription("OpenPepXL", "Identification");
```
Comment (timosachsenberg) [inline src/openms/source/APPLICATIONS/ToolHandler.cpp]: hmm I thought we had in our changelog that these are now tools..

File: src/openms/include/OpenMS/ANALYSIS/QUANTITATION/ProteinResolver.h (hunk 1 above)
Comment (cbielow) [reply src/openms/include/OpenMS/ANALYSIS/QUANTITATION/ProteinResolver.h]: yes. If you leave the doxygen comments like they were, all these parameters get attributed to the first active function below...

File: src/openms/source/APPLICATIONS/ToolHandler.cpp (hunk 2 above)
Comment (cbielow) [reply src/openms/source/APPLICATIONS/ToolHandler.cpp]: Ok, I did not know that. I used the minimal number of changes to make it consistent:
They had the 'bool official = false' set in TOPPBase and were still named '@page UTILS_..." in Doxygen, but were in the wrong category in ToolHandler. All of these need to be consistent for our docs to work properly.

Comment (timosachsenberg) [reply src/openms/source/APPLICATIONS/ToolHandler.cpp]: @enetz can you fix this?

Comment (enetz) [reply src/openms/source/APPLICATIONS/ToolHandler.cpp]: @timosachsenberg @cbielow It is very possible that I forgot some details when turning my tools from UTILs to TOOLs. Probably also applies to OpenPepXLLF and XFDR, not just OpenPepXL. I will look into it tomorrow.

**Comment by timosachsenberg** [comment]
@chris can you revert the downgrade to util and change the missing stuff to make it a tool?

**Comment by cbielow** [comment]
> @chris can you revert the downgrade to util and change the missing stuff to make it a tool?

what about UTILS_RNPxlSearch and UTILS_RNPxlXICFilter. They remain in UTIL?

**Comment by timosachsenberg** [comment]
yes that's fine as there will be a successor tool at some point

**Comment by cbielow** [comment]
Done.

Im attaching the remaining errors, if someone else wants to have a go... (this should really be a CI test...)
[doxygen-error.log](https://github.com/OpenMS/OpenMS/files/4230845/doxygen-error.log)

**Comment by jpfeuffer** [comment]
We have the doxygen logs nightly. I can add a parser for it maybe. I am a bit hesitant to add it to general CI as it takes quite some additional times.

**Comment by jpfeuffer** [comment]
You even see them in CDash as Warnings: http://cdash.openms.de/viewBuildError.php?type=1&buildid=401832

**Pull request merged**